templates/stake-banner.html

```diff
@@ -0,0 +1,14 @@
+<div class="stake-banner">
+  <div class="container">
+    <div class="banner-logo banner-logo-{{ CURRENT_LANGUAGE }}"></div>
+    <div class="row banner-content-row align-items-center justify-content-end">
+      <div class="col-lg-7 col-md-6 banner-content">
+        <h1>{{gettext("OGN Staking is live on OUSD.com")}}</h1>
+        <p>{{gettext("Stake your OGN to earn up to 25%%")}}</p>
+      </div>
+      <div class="col-md-3 banner-button">
+        <a class="btn btn-lg btn-primary" href="https://ousd.com/stake" target="_blank">Stake OGN now</a>
```

Comment:
Let's wrap this button for translation.

Reply:
Yep, I have added translation for the button and also added an env as "STAKE_URL" for the staking URL